hbase-server/src/main/java/org/apache/hadoop/hbase/mob/MobUtils.java

```diff
@@ -113,79 +111,71 @@ private MobUtils() {
 
   /**
    * Formats a date to a string.
+   *
    * @param date The date.
    * @return The string format of the date, it's yyyymmdd.
    */
-  public static String formatDate(Date date) {
-    return LOCAL_FORMAT.get().format(date);
+  public static String formatDate(LocalDate date) {
+    return DATE_TIME_FORMATTER.format(date);
+  }
+
+  public static String formatDate(long millisecond) {
+    return formatDate(toLocalDate(millisecond));
+  }
+
+  private static LocalDate toLocalDate(long millisecond) {
+    return Instant.ofEpochMilli(millisecond).atZone(ZoneId.systemDefault()).toLocalDate();
```

Comment:
Do we retain the same behavior with the old code? By default Date will use the system default time zone?

Reply:
Yes, the Date class just use the JVM timezone